app/controllers/verify_controller.rb

```diff
@@ -3,5 +3,20 @@ class VerifyController < ApplicationController
 
   check_or_render_not_found -> { IdentityConfig.store.idv_api_enabled }, only: [:show]
 
-  def show; end
+  def show
+    @app_data = app_data
+  end
+
+  private
+
+  def app_data
+    cipher = Encryption::AesCipher.encryption_cipher
+    session[:idv_api_store_key] ||= Base64.strict_encode64(cipher.random_key)
+    session[:idv_api_store_iv] ||= Base64.strict_encode64(cipher.random_iv)
+
+    {
+      store_key: session[:idv_api_store_key],
+      store_iv: session[:idv_api_store_iv],
```

Comment:
We'll need to use a unique IV for every encryption operation. We should generate a unique IV every time we perform encryption and store it alongside the ciphertext.

Reply:
> We'll need to use a unique IV for every encryption operation. We should generate a unique IV every time we perform encryption and store it alongside the ciphertext.

Updated in rebased 1d7409b1c.